ironfish-cli/src/commands/accounts/notes.ts

```diff
@@ -28,39 +28,46 @@ export class NotesCommand extends IronfishCommand {
 
     const client = await this.sdk.connectRpc()
 
-    const response = await client.getAccountNotes({ account })
```

Comment:
Is `getAccountNotes` unused now? If so, do we still need this method?

Reply:
The `getAccountNotes` method is unused now so I could delete the method. However, should I still leave in support for non-streaming calls to the `getAccountNotes` API? Could be helpful for future use cases, but I'm not sure if we would need to call this API anywhere else.

Reply:
I don't think we need the non-streaming implementation since notes are expected to grow overtime (unless we supported some filter or pagination). Similar to [exporting mined blocks](https://github.com/iron-fish/ironfish/blob/master/ironfish/src/rpc/routes/mining/exportMined.ts), we just stream all blocks that are returned. We can follow a similar pattern for account notes too.